src/MspFcComms.cpp

```diff
@@ -89,7 +99,8 @@ namespace FcComms
 
         MSP_RC rc_channels;
         sendMessage(rc_channels);
-        auto_pilot = static_cast<uint8_t>(rc_channels.getAutoEnabled());
+        auto_pilot = static_cast<uint8_t>(rc_channels.getAutoEnabled()); // To put on a ROS debug, right here and not from CommonFcComms
+        ROS_DEBUG("Autopilot status - %d", auto_pilot);
```

Comment:
Auto pilot status is already printed out in CommonFcComms.cpp when topics are published.